operator/pkg/operator.go

```diff
@@ -155,15 +155,21 @@ func (o *Operator) Start(ctx context.Context) error {
 				o.Logger.Infof("batch %x did not verify. Err: %v", newBatchLog.BatchMerkleRoot, err)
 				continue
 			}
-			responseSignature := o.SignTaskResponse(newBatchLog.BatchMerkleRoot)
+			batchIdentifier := append(newBatchLog.BatchMerkleRoot[:], newBatchLog.SenderAddress[:]...)
+			var batchIdentifierHash = *(*[32]byte)(crypto.Keccak256(batchIdentifier))
+
+			responseSignature := o.SignTaskResponse(batchIdentifierHash)
+			o.Logger.Infof("responseSignature about to send: %x", responseSignature)
 
 			signedTaskResponse := types.SignedTaskResponse{
+				BatchIdentifierHash: batchIdentifierHash,
 				BatchMerkleRoot: newBatchLog.BatchMerkleRoot,
+				SenderAddress:   newBatchLog.SenderAddress,
 				BlsSignature:    *responseSignature,
 				OperatorId:      o.OperatorId,
 			}
 
-			o.Logger.Infof("Signed hash: %+v", *responseSignature)
+			o.Logger.Infof("Signed Task Response to send: %+v", signedTaskResponse)
```

Comment:
<img width="714" alt="image" src="https://github.com/user-attachments/assets/deec3d91-3832-43f7-8bb3-520b94c59e3c">

We should hex encode relevant fields here, its hard to read